Analysis_of_rich_people.R

```diff
@@ -1,16 +1,70 @@
+# personName:             The full name of the billionaire.
+# age:                    The age of the billionaire in years.
+# country:                The country in which the billionaire resides.
+# city:                   The city in which the billionaire resides
+# source:                 The source of the billionaire's wealth (eg. company name).
+# industries:             The industries associated with the billionaire's business interests.
+# countryOfCitizenship:   The country of citizenship of the billionaire.
+# organization:           The name of the organization or company associated with the billionaire.
+# selfMade:               Indicates whether the billionaire is self-made (True/False).
+# status:                 D=Entrepreneur and U = Inherited
+# gender:                 The gender of the billionaire.
+# birthDate:              The birthdate of the billionaire.
+# lastName:               The last name of the billionaire.
+# firstName:              The first name of the billionaire.
+# title:                  The status or position of the billionaire (e.g., CEO, Founder).
+# date:                   The date of data collection.
+# state:                  The state in which the billionaire resides.
+# redidenceStateRegion:   The region or state of residence of the billionaire.
+# birthYear:              The birth year of the billionaire.
+# birthMonth:             The birth month of the billionaire.
+# birthDay:               The birth day of the billionaire.
+# cpi_country:            Consumer Price Index (CPI) for the billionaire's country.
+# cpi_change_country:     CPI change for the billionaire's country.
+# gdp_country:            Gross Domestic Product (GDP) for the billionaire's country.
+# gross_tertiary_education_enrollment:  
+#                         Enrollment in tertiary education in the billionaire's country.
+# gross_primary_education_enrollment_country: 
+#                         Enrollment in primary education in the billionaire's country.
+# life_expectancy_country:  
+#                         Life expectancy in the billionaire's country.
+# tax_revenue_country_country:  
+#                         Tax revenue in the billionaire's country.
+# total_tax_rate_country: Total tax rate in the billionaire's country.
+# population_country:     Population of the billionaire's country.
+# latitude_country:       Latitude coordinate of the billionaire's country.
+# longitude_country:      Longitude coordinate of the billionaire's country.
+
+# ------------------------------------------------------------------------------
+
+# We first started by creating a list of the top 10 richest billionaires:
 
 # Create a data frame with the top 10 billionaires
-top_10_billionaires <- df[1:10, ]
+top_10_billionaires <- df_billionaires[1:10, ]
 
-# Convert the 'finalWorth' column to numeric
+# Convert the 'finalWorth' column to numeric values
 top_10_billionaires$finalWorth <- as.numeric(gsub(",", "", top_10_billionaires$finalWorth))
```

Comment:
Die Zeile ergibt gar keinen sinn, oder ich verstehe was falsch.
![image](https://github.com/RauschRobin/R-analysis-of-billionaires/assets/70578359/6c585e07-d00a-4571-9fb2-688a3e8aacc4)
Die Daten sind vorher schon numerisch

Reply:
Das noch raus ballern, dann kannst durch HEHE
Wenn wir schon hier richtig reviewen, dann muss das auch durch